src/index.js

```diff
@@ -16,5 +16,8 @@ client.on('message', (message) => handleMessage(message, config, client.commands
 client.on('ready', () => client.user.setActivity(config.activity.name, { type: config.activity.type }));
 
 client.on('error', (error) => console.log(error));
+process.on('unhandledRejection', function (reason, p) {
```

Comment:
Is there a reason to not use an arrow function here?

Reply:
either way works, I'll switch it to an arrow function.